lib/stdlib/doc/src/timer.xml

```diff
@@ -69,6 +69,31 @@
   </datatypes>
 
   <funcs>
+    <func>
+      <name name="apply_after" arity="2" since=""/>
+      <fsummary>Spawn a process evaluating <c>erlang:apply(Function, [])</c>
+        after a specified <c>Time</c>.</fsummary>
+      <desc>
+        <p>Evaluates <c>spawn(erlang, apply, [<anno>Function</anno>,
+          []])</c> after <c><anno>Time</anno></c> milliseconds.</p>
```

Comment:
```suggestion
        <p>Evaluates <c>spawn(<anno>Function</anno>)</c>
          after <c><anno>Time</anno></c> milliseconds.</p>
```

Reply:
Hm, while that has the same outcome, `spawn(Function)` is _not_ what it does. It _does_ call `spawn(erlang, apply, [Function, []])`, as it currently says.

Reply:
OK.